flask_sqlalchemy/__init__.py

```diff
@@ -202,7 +209,13 @@ def get_bind(self, mapper=None, clause=None):
             if bind_key is not None:
                 state = get_state(self.app)
                 return state.db.get_engine(self.app, bind=bind_key)
-        return SessionBase.get_bind(self, mapper, clause)
+        return super().get_bind(
```

Comment:
2.x still supports python 2, so the super shorthand won't work

Reply:
got it!